Docs/Api/CSharpDriverDocs.shfbproj

```diff
@@ -58,6 +58,7 @@
       <NamespaceSummaryItem name="MongoDB.Driver.Core.Clusters" isDocumented="True">The MongoDB.Driver.Core.Clusters namespace contains classes related to clusters.</NamespaceSummaryItem>
       <NamespaceSummaryItem name="MongoDB.Driver.Core.Clusters.ServerSelectors" isDocumented="True">The MongoDB.Driver.Core.Clusters.ServerSelectors namespace contains classes related to server selectors.</NamespaceSummaryItem>
       <NamespaceSummaryItem name="MongoDB.Driver.Core.Configuration" isDocumented="True">The MongoDB.Driver.Core.Configuration namespace contains classes related to configuration.</NamespaceSummaryItem>
+      <NamespaceSummaryItem name="MongoDB.Driver.Core.Compression" isDocumented="True">The MongoDB.Driver.Core.ConnecitonPools namespace contains classes related to compression.</NamespaceSummaryItem>
```

Comment:
>ConnecitonPools 

typo?

Reply:
Also, could you, please describe the idea of these `NamespaceSummaryItem` items?

>The MongoDB.Driver.Core.ConnecitonPools namespace contains classes related to compression.<

is it realy true?

Reply:
No it is not correct. After correcting typos the correct description is:

    The MongoDB.Driver.Core.Compression namespace contains classes related to compression.

Reply:
`NamespaceSummaryItems` are required by Sandcastle to describe namespaces.